src/ResourceManager/ManagedServiceIdentity/Commands.ManagedServiceIdentity/help/Get-AzureRmUserAssignedIdentity.md

```diff
@@ -0,0 +1,229 @@
+---
+external help file: Microsoft.Azure.Commands.ManagedServiceIdentity.dll-Help.xml
+Module Name: AzureRM.ManagedServiceIdentity
+online version: 1.0.0
```

Comment:
online version should follow this pattern: 
online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.eventhub/new-azurermeventhub

Fix for all three help files

Reply:
Are you referring to the format of the file or the numeric value of online version?

Reply:
@varunkch She's saying it should be a link to the document. `online version` doesn't mean a version number. It means an online version of the help file. So, her example of the link format is:

```
online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.eventhub/new-azurermeventhub
```

Reply:
See Yanni's comment.  If you have questions about this please ping me.

Reply:
We don't have an online version of the help file yet. Is it possible to translate this md help file to an online version?

Reply:
All you need to do is follow the format, and it will create the proper link:

```
online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.managedserviceidentity/get-azurermuserassignedidentity

online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.managedserviceidentity/new-azurermuserassignedidentity

online version: https://docs.microsoft.com/en-us/powershell/module/azurerm.managedserviceidentity/remove-azurermuserassignedidentity
```